uportal-war/src/main/java/org/jasig/portal/rest/SessionRESTController.java

```diff
@@ -81,6 +88,13 @@ public ModelAndView isAuthenticated(HttpServletRequest request, HttpServletRespo
             attributes.put("displayName", person.isGuest() ? "Guest" : person.getFullName());
             attributes.put("sessionKey", person.isGuest() ? null: key); //only provide keys to non guest users
             attributes.put("version", uPortalVersion);
+            if(swapperManager != null) {
```

Comment:
Doesn't this approach create a race condition?  IIRC you can't count on the order in which dependencies are injected into beans by the app context.  Would it be better to handle this task in a @PostConstruct initialization method?